### PR TITLE
Add support for beiboot-via-sudo

### PR DESCRIPTION
**Pull request by allisonkarlitskaya**
This is the last missing piece from Cockpit Client beibooting — we can now run a root bridge, recursively beibooted.

### CODE REVIEW

**Comment by martinpitt** [comment]
This generally looks good to me, thanks! The `AttributeError: 'TestBridge' object has no attribute 'transport'` is breaking a lot. Also, can you please rebase so that tmt tests have a better chance of working?

File: src/client/cockpit-beiboot [hunk 1]
```diff
@@ -220,8 +223,14 @@ class SshPeer(Peer):
         logger.debug("Launching command: cmd=%s env=%s", cmd, env)
         transport = await self.spawn(cmd, env, stderr=agent, start_new_session=True)
 
+        if not self.always:
+            usr_bin_cockpit_bridge_steps = [('try_exec', (['/usr/bin/cockpit-bridge'],))]
```
Comment (martinpitt) [inline src/client/cockpit-beiboot:227]: Could we please drop the hardcoded path? `sudo make install` installs into /usr/local, and we even used to do that in our /pybridge scenario. This ought to work with `$PATH`, no?

Or will this conflict with the flatpak and /apps/ etc.? Is that in $PATH?

File: src/client/cockpit-beiboot [hunk 2]
```diff
@@ -283,6 +297,7 @@ def main() -> None:
     polyfills.install()
 
     parser = argparse.ArgumentParser(description='cockpit-bridge is run automatically inside of a Cockpit session.')
+    parser.add_argument('--always', action='store_true', help="Never try to run /usr/bin/cockpit-bridge")
```
Comment (martinpitt) [inline src/client/cockpit-beiboot:300]: "installed bridge"?

**Review by martinpitt — CHANGES_REQUESTED** [review]

File: src/client/cockpit-beiboot [hunk 3]
```diff
@@ -280,6 +281,9 @@ async def run(args) -> None:
 
     try:
         message = await bridge.ssh_peer.start()
+        # https://github.com/cockpit-project/cockpit/issues/18927
+        if 'capabilities' in message:
```
Comment (mvollmer) [inline src/client/cockpit-beiboot:273]: Should this also check that this is a "init" message?

**Pull request merged**